src/main/java/org/rsa/listeners/AutoCompleteListener.java

```diff
@@ -15,8 +18,14 @@ public void onCommandAutoCompleteInteraction(@NotNull CommandAutoCompleteInterac
         }
 
         CommandObject command = Commands.getCommand(event.getName());
-        if (command.isAutocomplete()) {
+        if (Objects.nonNull(command) && command.isAutocomplete()) {
             command.onCommandAutoCompleteInteraction(event);
+            return;
+        }
+
+        CommandObjectV2 commandObjectV2 = Commands.getCommandV2(event.getName());
+        if (Objects.nonNull(commandObjectV2) && commandObjectV2.isAutocomplete()) {
+            commandObjectV2.onCommandAutoCompleteInteraction(event);
         }
```

Comment:
Should add a check if the event has not been replied to and reply with an empty list.